Exploration Queries/InputEntity_Account/UserAccount_Peers.yaml

```diff
@@ -0,0 +1,41 @@
+Id: 18B7E4E3-5B57-4924-B3CD-7E9A5A143521
+DisplayName: Account's peers involved in alerts
+Description: The query located the Account's peers that were involved in a recent alert 
```

Comment:
Maybe just put this in description - "Locates the Account's peers with a recent alert"

Reply:
done